# empty request
---
uint32 distanceA
uint32 distanceB
uint32 distanceC
